contracts/RMRK/RMRKEquippable.sol

```diff
@@ -204,28 +170,8 @@ contract RMRKEquippable is RMRKNesting, IRMRKEquippable {
         return getResource(resourceId);
     }
 
-    function getFullResources(uint256 tokenId)
```

Comment:
Removing these additional view functions really keeps the main contract lean, I like it 👍

Reply:
Yeah! We should do that on the EIP too.